src/sentry/integrations/opsgenie/integration.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+import logging
+from typing import Any, Mapping, Sequence
+
+from django.utils.translation import gettext_lazy as _
+
+from sentry.integrations.base import (
+    FeatureDescription,
+    IntegrationFeatures,
+    IntegrationInstallation,
+    IntegrationMetadata,
+    IntegrationProvider,
+)
+from sentry.pipeline import PipelineView
+
+logger = logging.getLogger("sentry.integrations.opsgenie")
+
+DESCRIPTION = """
+Trigger alerts in Opsgenie from Sentry.
+
+OpsGenie is a cloud-based service for dev & ops teams, providing reliable alerts, on-call schedule management and escalations.
+OpsGenie integrates with monitoring tools & services and ensures that the right people are notified via email, SMS, phone calls,
```

Comment:
It looks like `Opsgenie` only ever capitalizes the O, so we should keep that consistent